en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+# FHIR Connector Example
+
+Given below is a sample scenario that demonstrates how to work with resources in [FHIR](https://www.hl7.org/fhir/overview.html)(Fast Healthcare Interoperability Resources). FHIR specification is a standard for exchanging health care information electronically.The WSO2 EI FHIR connector uses the FHIR RESTFul API to interact with FHIR.
+
```

Comment:
Shall we put a link to the API or the API doc? 
What version of API we support? 
What the connector is for : "The connector can be used to invoke FHIR operations within WSO2 EI mediation logic"

Reply:
Added API docs and description to the doc.